src/components/Section.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Section.scss';
+
+function Section() {
```

Comment:
The section does not need a section class because it's the shares the same styling as all other sections.